analytics/filesystem/file_module_test.go

```diff
@@ -8,12 +8,25 @@ import (
 
 	"github.com/prebid/prebid-server/v2/analytics"
 	"github.com/prebid/prebid-server/v2/config"
+	"github.com/stretchr/testify/mock"
 
 	"github.com/prebid/openrtb/v20/openrtb2"
 )
 
 const TEST_DIR string = "testFiles"
 
+type MockLogger struct {
+	mock.Mock
+}
+
+func (mw *MockLogger) Debug(v ...interface{}) {
+	mw.Called(v)
+}
+
+func (mw *MockLogger) Flush() {
```

Comment:
Nitpick: why is this named `mw` as opposed to `ml`?

Reply:
fixed